{
  try {
    setId(id);
    setName(name);
    setValue(value);
  }
 catch (  Exception e) {
    throw new CreateException("Exception occurred: " + e);
  }
  return null;
}
